docs/source/dev_guide/algorithms/k_nearest_neighbors/k-nearest-neighbors-knn-classifier.rst

```diff
@@ -68,9 +76,19 @@ Feature vectors from any bucket belong to the region of the space
 defined by tree nodes on the path from the root node to the
 respective leaf.
 
+Brute Force
+-----------
+
+Brute Force algorithm calculates the squared distances from each query feature vector
+to each reference feature vector in the training data set. Then,
+for each query feature vector it selects :math:`k` objects from the training set that are closest to that query feature vector.
+
```

Comment:
Reference to bibliography is required.

Reply:
I think Alexey [answered to the wrong comment[(https://github.com/oneapi-src/oneDAL/pull/667#discussion_r448949701) that there is no article to refer to

Reply:
Nobody said that it should be single reference and/or it should cover all aspects of our implementation. But absense of reference at all is not consistent.

Reply:
Added references